src/ray/protobuf/direct_actor.proto

```diff
@@ -33,8 +33,20 @@ message PushTaskReply {
   repeated ReturnObject return_objects = 1;
 }
 
+message DirectActorCallArgWaitCompleteRequest {
```

Comment:
This would also be very useful for @suquark s ray `async_api`. Using it could allow us to get rid of the last remaining python plasma client in ray :)